change_problem.py

```diff
@@ -23,45 +23,63 @@
 matplotlib.use("agg")
 import matplotlib.pyplot as plt
 
+# TODO: Enter your token here
+token = 'YOUR-TOKEN-HERE'
+
 # Set the solver we're going to use
-from dwave.system.samplers import DWaveSampler
-from dwave.system.composites import EmbeddingComposite
+def set_sampler():
+    from dwave.system.samplers import DWaveSampler
+    from dwave.system.composites import EmbeddingComposite
+
+    sampler = EmbeddingComposite(DWaveSampler(endpoint='https://cloud.dwavesys.com/sapi/', token=token, solver={'qpu': True}))
+
+    return sampler
+
+def create_graph():
+    # Create empty graph
+    G = nx.Graph()
+
+    ## TODO:  Update graph to new problem graph
+
+    return G
 
-sampler = EmbeddingComposite(DWaveSampler(solver={'qpu': True}))
+def solve_problem(G, sampler):
 
-# Create empty graph
-G = nx.Graph()
+    ## TODO:  Update dwave-networkx function to new problem function
 
-# Add edges to graph - this also adds the nodes
+    return 
```

Comment:
I'm going to leave return here instead of pass even though it's non-standard; we won't have discussed pass with the students at all in the class and it might add to more off-topic confusion.